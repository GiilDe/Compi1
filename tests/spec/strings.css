"simple string #1"
'simple string #2'
"many tagss'''''''''"
'many quotes """" '
"h\n\te\n\t\tl\n\t\t\tl\n\t\t\t\to\n"
'\nh\n\te\n\t\tl\n\t\t\tl\n\t\t\t\to\n'
""
''
'\a'
'\aa'
'\aaa'
'\aaaa'
'\aaaaa'
'\aaaaaa'
'\aaaaaaa'
'\\n'
'\3a'
'\3A'
'\3a\29'
'\3A\29'
'not\20\19legal\20\7Fletters'
'\\this\\is"\63ompletely\\healthy"'
"IF YOU GOT HERE WELL DONE :D"
"and this will fuck it up a bit \GOODLUCK"
