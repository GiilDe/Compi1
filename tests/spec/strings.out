1 STRING simple string #1
2 STRING simple string #2
3 STRING many tagss'''''''''
4 STRING many quotes """"
5 STRING h\n\te\n\t\tl\n\t\t\tl\n\t\t\t\to\n
6 STRING
h
	e
		l
			l
				o

7 STRING
8 STRING
9 STRING
10 STRING
11 STRING
12 STRING
13 STRING
14 STRING
15 STRING a
16 STRING \n
17 STRING :
18 STRING :
19 STRING :)
20 STRING :)
21 STRING not legal letters
22 STRING \this\is"completely\healthy"
23 STRING IF YOU GOT HERE WELL DONE :D
Error undefined escape sequence G
